### DISCUSSION TITLE
Training time or Testing time

### DISCUSSION THREAD

**Discussion by chiru7187** [pycaret/pycaret#3862] [Q&A]
what is that TT(sec) represents during compare_models?

**Comment by chiru7187** [comment]
Usually it is taking more than the time showing in TT(sec) during training ...I got confussed is it time for itertion or epoch or for entire duration?

**Comment by ngupta23** [comment]
It is training time per fold. If you are running 10 folds, actual clock time will be > 10x that time due to other overhead.

**Comment by chiru7187** [comment]
Thank you for your response. If I want to know total training time
including other overhead, how to know it

On Thu, 21 Dec 2023 at 8:23 PM, Nikhil Gupta ***@***.***>
wrote:

> It is training time per fold. If you are running 10 folds, actual clock
> time will be > 10x that time due to other overhead.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pycaret/pycaret/discussions/3862#discussioncomment-7919141>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANQYLGHQZGSDQSIOT6JAK5TYKREODAVCNFSM6AAAAABA6JD3GOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TSMJZGE2DC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Chiranjeevi Yarramsetty
Research Scholar,
Department of Electrical & Electronics Engineering,
National Institute of Technology Karnataka, Surathkal, Srinivasa Nagar,
Mangalore - 575025.
Karnataka, INDIA.
Mobile No: +91 8328566282
                          9533519351

https://orcid.org/0000-0002-3833-1644
